README.md

```diff
@@ -203,7 +205,26 @@ notably `case` and `let`:
     • In an equation for ‘f’: f x = let y = x in y
 ```
 
-Other than that, pattern synonyms are also not compatible with linear types
+### Records
+Record syntax itself works fine, but please note that the constructors are fully
+linear; this is mainly because mixed-linearity records would need a new and
```

Comment:
Since "full linearity" is the default for constructors. It's not necessarily useful to announce it upfront.

The point, basically, is that the type of projections is unchanged (takes an unrestricted record) because a linear projection would be incorrect in general. You can add that we could special-case the "one-field record" since it may be useful to define convenient `newtype`-s.

If you want to push it, you could say that the most general form is that the projection from the only linear field of a record can be linear, but that there is no syntax at the time for unrestricted record fields.